stores/blocks/src/main/java/com/bloxbean/cardano/yaci/store/blocks/storage/impl/BlockCborStorageImpl.java

```diff
@@ -0,0 +1,58 @@
+package com.bloxbean.cardano.yaci.store.blocks.storage.impl;
+
+import com.bloxbean.cardano.yaci.store.blocks.BlocksStoreProperties;
+import com.bloxbean.cardano.yaci.store.blocks.domain.BlockCbor;
+import com.bloxbean.cardano.yaci.store.blocks.storage.BlockCborStorage;
+import com.bloxbean.cardano.yaci.store.blocks.storage.impl.model.BlockCborEntity;
+import com.bloxbean.cardano.yaci.store.blocks.storage.impl.repository.BlockCborRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Slf4j
+public class BlockCborStorageImpl implements BlockCborStorage {
+
+    private final BlockCborRepository blockCborRepository;
+    private final BlocksStoreProperties blocksStoreProperties;
+
+    @Override
+    @Transactional
+    public void save(BlockCbor blockCbor) {
+        if (!blocksStoreProperties.isSaveCbor()) {
```

Comment:
@ducpm2303 I think we can keep this check in processor layer and keep storage layer simple.
So, if in the future, we introduce key value based blockCborStorage, we don't need to do any duplicate check. Same for delete method for rollback.